go.mod

```diff
@@ -153,3 +160,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
+
+replace github.com/fluxcd/source-controller/api => github.com/dipti-pai/source-controller/api v0.0.0-20240913204118-48417bdc8bd7
```

Comment:
You can remove this and use the main branch from SC.

Reply:
Do we need to do a release of the SC branch from fluxcd:main ? If I remove the replace, source-controller api version v1.3.0 from the May release is being referred which does not have the provider field. Thanks.

Reply:
for now you can do `go get github.com/fluxcd/source-controller/api@main`

Reply:
Hmm, I tried that however make is failing as its unable to find the file with the new CRD. If I manually try to get the file, I get a "404: Not Found". Am I missing something else ? 

```
curl -s --fail https://raw.githubusercontent.com/fluxcd/source-controller/v1.3.1-0.20240916104646-ccdd523e06d1/config/crd/bases/source.toolkit.fluxcd.io_gitrepositories.yaml \
        -o cache/gitrepositories_v1.3.1-0.20240916104646-ccdd523e06d1.yaml
make: *** [Makefile:98: cache/gitrepositories_v1.3.1-0.20240916104646-ccdd523e06d1.yaml] Error 22
```